gradle-plugin/src/test/resources/javaOutput/search-micronaut/SearchResource.java.txt

```diff
@@ -44,7 +45,7 @@ public class SearchResource {
 
   @Get
   @Produces("application/json")
-  public SearchGetResourceModel getEntity(@NotNull @Size(min = 1) @QueryValue String q,
+  public SearchGetResourceModel getEntity(@NotNull @Size(min = 1) @QueryValue List<String> q,
```

Comment:
Theres a problem with the @Size annotation when providing a list. It was used to check that the string has at least one letter in it.